.github/workflows/build-test-package.yml

```diff
@@ -3,270 +3,14 @@ name: Build, test, package
-#        include:
-#          - itk-python-git-tag: "v5.3rc04"
-#
-#    steps:
-#      - uses: actions/checkout@v4
-#        with:
-#          path: "im"
-#
-#      - name: 'Install Python'
-#        run: |
-#          $pythonArch = "64"
-#          $pythonVersion = "3.${{ matrix.python-version-minor }}"
-#          iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/scikit-build/scikit-ci-addons/master/windows/install-python.ps1'))
-#
-#      - name: 'Fetch build dependencies'
-#        shell: bash
-#        run: |
-#          mv im ../../
-#          cd ../../
-#          curl -L "https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${{ matrix.itk-python-git-tag }}/ITKPythonBuilds-windows.zip" -o "ITKPythonBuilds-windows.zip"
-#          7z x ITKPythonBuilds-windows.zip -o/c/P -aoa -r
-#          curl -L "https://data.kitware.com/api/v1/file/5c0ad59d8d777f2179dd3e9c/download" -o "doxygen-1.8.11.windows.bin.zip"
-#          7z x doxygen-1.8.11.windows.bin.zip -o/c/P/doxygen -aoa -r
-#          curl -L "https://data.kitware.com/api/v1/file/5bbf87ba8d777f06b91f27d6/download/grep-win.zip" -o "grep-win.zip"
-#          7z x grep-win.zip -o/c/P/grep -aoa -r
-#
-#      - name: 'Build 🐍 Python 📦 package'
-#        shell: cmd
-#        run: |
-#          cd ../../im
-#          call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
-#          set PATH=C:\P\grep;%PATH%
-#          set CC=cl.exe
-#          set CXX=cl.exe
-#          C:\Python3${{ matrix.python-version-minor }}-x64\python.exe C:\P\IPP\scripts\windows_build_module_wheels.py --py-envs "3${{ matrix.python-version-minor }}-x64"
-#
-#      - name: Publish Python package as GitHub Artifact
-#        uses: actions/upload-artifact@v1
-#        with:
-#          name: WindowWheel3.${{ matrix.python-version-minor }}
-#          path: ../../im/dist
-#
-#  publish-python-packages-to-pypi:
-#    needs:
-#      - build-linux-python-packages
-#      - build-macos-python-packages
-#      - build-windows-python-packages
-#    runs-on: ubuntu-18.04
-#
-#    steps:
-#      - name: Download Python Packages
-#        uses: actions/download-artifact@v2
-#
-#      - name: Prepare packages for upload
-#        run: |
-#          ls -R
-#          for d in */; do
-#            mv ${d}/*.whl .
-#          done
-#          mkdir dist
-#          mv *.whl dist/
-#          ls dist
-#
-#      - name: Publish 🐍 Python 📦 package to PyPI
-#        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-#        uses: pypa/gh-action-pypi-publish@master
-#        with:
-#          user: __token__
-#          password: ${{ secrets.pypi_password }}
+  cxx-build-workflow:
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.2
+    with:
+      cmake-options: "Module_Montage_BUILD_EXAMPLES:BOOL=ON"
+
+  python-build-workflow:
```

Comment:
Removing this section should remove attempts of creating the packages. The python package section of the old file was commented out. Maybe do the same here?

Reply:
Added the logic from 

  with:
      warnings-to-ignore: 'ld. warning. ignoring duplicate libraries. \'-lm\''